src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,89 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private int cilindersNumber;
+    private int volumeCilinders;
+    private int weightEngine;
+    private String engineType;
+    private String kppType;
 
+    public Engine(int cilindersNumber,
+                  int volumeCilinders,
+                  int weightEngine,
+                  String engineType,
+                  String kppType) {
```

Comment:
it is better to wrap text onto next line only if its length is more than 100 symbols
please make two lines here

Reply:
Fixed.